topi/python/topi/_ffi/libinfo.py

```diff
@@ -0,0 +1,79 @@
+"""Library information."""
+from __future__ import absolute_import
```

Comment:
it is likely we don't need these ffi utilities, but can simply reuse all things under TVM.
The find lib path could be an exception but it is also fine if we give a search_path.

Reply:
if there is a need to enhance tvm's ffi utilities, let us do so